src/kitovu/config.py

```diff
@@ -1,16 +1,22 @@
 """Storage and parsing of kitovu's configuration."""
 
 import typing
-import urllib.parse
+import getpass
 
 import keyring
 
 
-def get_password(url: str) -> str:
-    """Get the password for the given URL via keyring."""
-    # FIXME Handle password being None
-    # FIXME Make sure there's an username
-    assert urllib.parse.urlparse(url).username is not None
-    password: typing.Optional[str] = keyring.get_password('kitovu', url)
-    assert password is not None
+def get_password(plugin: str, identifier: str) -> str:
+    """Get the password for the given URL via keyring.
+
+    Args:
+       plugin: The name of the plugin requesting a password.
+       identifier: An unique identifier (such as an URL) for the connection.
+    """
+    service = f'kitovu-{plugin}'
+    password: typing.Optional[str] = keyring.get_password(service, identifier)
+    if password is None:
+        # FIXME handle this in a nicer way
+        password = getpass.getpass()
+        keyring.set_password(service, identifier, password)
```

Comment:
This is a quick & dirty hack to make it ask for a password if none is set - so now you should be able to do e.g. `kitovu sync smb fbruhin` and enter your password to test it out!

Reply:
so "if password is None": in the end you just enter a bogus password? Or you take what's already there?

Reply:
`getpass.getpass()` asks the user for a password on the commandline ([docs](https://docs.python.org/3/library/getpass.html)) - so we ask the user, then store the password in the keyring (for the given plugin and user), and then use the password they entered.